mandos-go/json/parse/parseAccounts.go

```diff
@@ -35,6 +35,17 @@ func (p *Parser) processAccount(acctRaw oj.OJsonObject) (*mj.Account, error) {
 			if err != nil {
 				return nil, fmt.Errorf("invalid account comment: %w", err)
 			}
+		case "shard":
+			acct.Shard, err = p.processUint64(kvp.Value)
+			if err != nil {
+				return nil, fmt.Errorf("invalid shard number: %w", err)
+			}
+		case "isSmartContract":
```

Comment:
should be determined based on whether or no the account has "code"
fixed in another PR, will merge there